source/administration/config-settings.rst

```diff
@@ -1300,6 +1321,14 @@ The attribute in the SAML Assertion that will be used to populate the username f
 | This feature's ``config.json`` setting is ``"UsernameAttribute": ""`` with string input.                                                                             |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Id Attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) The attribute in the SAML Assertion that will be used to bind users from SAML to users in Mattermost.
```

Comment:
`that will be used to` --> `used`

Why/when would an admin use this setting? (Might be explained in the other SAML/LDAP sync doc update)